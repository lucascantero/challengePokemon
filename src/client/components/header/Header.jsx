import React from 'react';
import ButtonLanguage from '../buttonLanguage/ButtonLanguage'

const Header = () => {
 
    return(
        <div className="searchBox__contenedor">
            <ButtonLanguage/>
        </div>
    )
}

export default Header;