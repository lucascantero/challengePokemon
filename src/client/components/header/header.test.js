import React from 'react';
import Header from './Header.jsx';
import { act } from 'react-dom/test-utils';
import {render,unmountComponentAtNode} from 'react-dom';


describe('Testing Header', () => {

    let container = null;

    beforeEach(() => {
        container = document.createElement("div");
        document.body.appendChild(container);
    });

    afterEach(() => {
        unmountComponentAtNode(container);
        container.remove();
        container = null;
    });

    it('Should match snapshot', () => {
        
        act(() => {
            render(<Header t={key => key} />, container)
        })

        expect(container.innerHTML).toMatchSnapshot();
    })
})