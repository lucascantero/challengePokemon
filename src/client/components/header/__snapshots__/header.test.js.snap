// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Should match snapshot 1`] = `"<div class=\\"searchBox__contenedor\\"><div class=\\"buttonLanguage__wrapper\\"><button class=\\"buttonLanguage__button\\">en</button><button class=\\"buttonLanguage__button\\">es</button></div></div>"`;
